zilliqa/src/consensus.rs

```diff
@@ -37,9 +37,8 @@ use crate::{
 
 #[derive(Debug)]
 struct NewViewVote {
-    signatures: Vec<NodeSignature>,
-    signers: Vec<u16>,
-    cosigned: BitVec,
+    // Keeps signer index in deposit contract and sent signature
+    signatures: Vec<(u16, NodeSignature)>,
```

Comment:
Given we need to convert this to a bitvec later when we construct the aggregate QC and given it will generally be quite a dense list, why not just use a bitvec here too? Especially since we're sorting the index list later.